src/generated/data/dictionary-profanity.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
Why not in the assets submodule?

Reply:
Because only Teryl will use this, it's specific to words Owlbot has in its database.